enhancements/update/manifest-install-levels.md

```diff
@@ -0,0 +1,196 @@
+The default value is `0`, and currently the only other allowed value is `1`.
+
+The cluster-version operator has [three reconciliation phases][manifest-graph]:
+
+* Install phase, which lasts until all release image manifests have been reconciled.
+    This enhancement is about the timing of these manifests.
+* Update phase, which occurs after installation if the desired release image has not yet been fully reconciled.
+    This enhancement has nothing to do with this phase.
+* Reconcile phase, which occurs when neither the installation nor the update phase applies.
+    This enhancement has nothing to do with this phase.
+
+The cluster-version operator will only consider the new annotation during the install phase, and will ignore it for all other phases.
+
+To guard against errors in release image manifests, the install-phase cluster-version operator will add the following manifests guards:
+
+* The value of the new `release.openshift.io/install-level` annotation must be `true`.
+* If the new `release.openshift.io/install-level` annotation is present, [the create-only annotation][create-only] must also be present.
+
+If a manifest fails any install-phase, valid-manifest guards, the cluster-version operator will complain about the invalid value in a `Failing=True` ClusterVersion condition and not apply any release image manifests.
+That ensures that installs will fail with an understandable explanation, making it very unlikely that accidentally mistyped values make it through CI and into released images.
+
+During the install phase, the cluster-version operator will ensure all manifests in phase *n* are reconciled before attempting to apply any manifests from phase *n*+1.
+Within a phase, manifests will still be applied in parallel without blocking other manifests within that same phase.
+
+Some component manifests are required to satisfy bootstrap completion, which [today is][bootstrap-complete]:
+
+* Production pods for:
+    * `openshift-cluster-version/cluster-version-operator`
+    * `openshift-kube-apiserver/kube-apiserver`
+    * `openshift-kube-controller-manager/kube-controller-manager`
+    * `openshift-kube-scheduler/openshift-kube-scheduler`
+* A bootstraped etcd cluster.
+
+Manifests which are required for bootstrap completion must remain in the default level `0`, ideally by leaving the new annotation unset.
+
+At least one manifest which is fairly slow, ideally always landing after bootstrap removal must remain in the default level `0`, ideally by leaving the new annotation unset.
+For example, the monitoring ClusterOperator always takes a while, especially when it is scheduled to compute nodes.
+FIXME: but when we allow monitoring to be scheduled on the control plane, is it _always_ after bootstrap removal?  On the other hand, all we really need to beat is cluster-bootstrap's manifest-pushing phase, which completes well before its pod-waiting phase.
```

Comment:
Change `ideally always` to either `ideally` or `always`

`FIXME: but when we allow monitoring to be scheduled on the control plane, is it _always_ after bootstrap removal?  On the other hand, all we really need to beat is cluster-bootstrap's manifest-pushing phase, which completes well before its pod-waiting phase.`  Not sure what you want to communicate here. Also this should be a separate paragraph.

Reply:
I added a comma with a70584d -> e792bbf, to make it more clear that "ideally always landing after bootstrap removal" is a qualifier for "fairly slow".  It doesn't have to always land after bootstrap removal, but that is ideal.  "ideally landing after bootstrap removal", without the "always" does not remind folks that these pushes are racy, and landing after bootstrap removal once does not mean that manifest will land after bootstrap removal on all subsequent installs.